### DISCUSSION TITLE
Home-Assistant integration

### DISCUSSION THREAD

**Discussion by Disane87** [Donkie/Spoolman#123] [General]
Currently I'm working on a custom component to integrate Spoolman into Home-Assistant. Home-Assistant is a SmartHome system, where you can automate things and so on.

I'm pretty far along in development and wanted to ask if you could give me the logo as a transparent PNG (without lettering) for it, or if I could use it for it myself?

When I have a certain state, I would also like to share the integration with you. Maybe you can include it here as a nice feature.

Currently this is the progress:
![image](https://github.com/Donkie/Spoolman/assets/956158/5fe0e8d4-1491-45c5-9e72-acebb8bb81be)

![image](https://github.com/Donkie/Spoolman/assets/956158/fb48722d-240c-4e98-9398-387f7d9dff04)

**Comment by Donkie** [comment]
Very nice! I use HA myself so I know it.

In the client/icons folder you have the icon sources. You should be able to use the svgs directly, if not you can just find some tool to render them to a png.

**Comment by Disane87** [comment]
Thank you! Will create a repo soon. If you want, you can beta test it and give some valuable feedback.

**Comment by Donkie** [comment]
Yes that would be nice :)

**Comment by Disane87** [comment]
Just released the first preview of the integration: https://github.com/Disane87/spoolman-homeassistant
It's my first Python and HA-Integration project :D So please don't be mad.

You can raise up an issue if you're something missing or if something is broke.

You can install it via HACS and add my repo to custom repos. Will update the readme soon to reflect that.

**Reply by Disane87** [reply]
I also have submitted your logo and icon to the HA brand repo to be available in HA
